src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/AWSTracingPipelineCustomizer.cs

```diff
@@ -44,6 +44,6 @@ public void Customize(Type serviceClientType, RuntimePipeline pipeline)
             return;
         }
 
-        pipeline.AddHandlerAfter<EndpointResolver>(new AWSTracingPipelineHandler(this.options));
+        pipeline.AddHandlerAfter<RetryHandler>(new AWSTracingPipelineHandler(this.options));
```

Comment:
"The fix is for OpenTelemetry to inject itself before the RetryHandler"

But this is injecting AFTER. Which is correct? The summary or the code in this case?

Reply:
Fixed, good catch.